### PR TITLE
Feature/like function

### PR DESCRIPTION
**Pull request by niklasrammelkamp**
Hi, here's my pull request to the following User Story:
[like function](https://github.com/niklasrammelkamp/capstone-project/issues/14#issue-1558134816)

### CODE REVIEW

File: components/ButtonLike/index.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        headers: {
+          "Content-Type": "application/json",
+        },
+      });
+
+      if (response.ok) {
+        const data = await response.json();
+      } else {
+        console.error(`Error: ${response.status}`);
+      }
```
Comment (AGeck113) [inline components/ButtonLike/index.js:20]: Hi Niklas,
do you need "data" here? It looks like you are not doing anything with it.
Maybe you could use something like that?
`if (!response.ok) {
console.error(`Error: ${response.status}`)
}`

File: components/ButtonLike/index.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
```
Comment (AGeck113) [inline components/ButtonLike/index.js:41]: same as above

File: components/ButtonLike/index.js (hunk 1 above)
Comment (niklasrammelkamp) [reply components/ButtonLike/index.js:20]: True, thank you :)

File: components/ButtonLike/index.js (hunk 2 above)
Comment (niklasrammelkamp) [reply components/ButtonLike/index.js:41]: True, thank you :)

**Review by AGeck113 — APPROVED** [review]

**Pull request merged**